pkg/vpcmodel/grouping.go

```diff
@@ -129,13 +146,14 @@ func (g *GroupConnLines) groupExternalAddresses() {
 				continue
 			}
 			connString := conns.String()
+			connIsStateful := fmt.Sprintf("%s;;%d", connString, conns.IsStateful)
```

Comment:
Variable name is a bit misleading. How about `connWithStatefulness`?

Reply:
Done, also in other places I used this (or a very similar) name